packages/plugins/sentry/src/index.ts

```diff
@@ -179,6 +179,8 @@ export const useSentry = (options: SentryPluginOptions = {}): Plugin => {
         Sentry.configureScope(scope => options.configureScope!(args, scope));
       }
 
+      extendContext({ sentry: { span: rootSpan } });
```

Comment:
```suggestion
      extendContext({ sentry: { scope } });
```

Also, I believe this is rather what should be set as it is the _Scope_ rather than the _Span_ that's Sentry's concrept of a context.

Reply:
Didn't mean for the diff to be directly applied 😅 

But I feel like with this, `Sentry.configureScope(scope => scope.setSpan(rootSpan));` would go outside of the if-else